src/main/java/duke/logic/commands/results/CommandResultCalender.java

```diff
@@ -2,6 +2,9 @@
 
 import duke.model.lists.EventList;
 
+/**
+ * Class representing the result of any calender command.
```

Comment:
> Interface name does not suggest that is exclusive to command result, so command result should be omitted from the comments.

As mentioned above, over here then mention the command result

Reply:
alright